pkg/daemon/ceph/osd/volume.go

```diff
@@ -677,6 +683,15 @@ func (a *OsdAgent) initializeDevicesLVMMode(context *clusterd.Context, devices *
 					}...)
 				}
 
+				// assign the initial weight specific to the device
+				initialWeight := device.Config.InitialWeight
+				if initialWeight != "" {
+					immediateExecuteArgs = append(immediateExecuteArgs, []string{
```

Comment:
Is the initial weight needed both during the ceph-volume provisioining of the OSD and the OSD daemon? I would have thought it would only be needed in one place or the other, but not both.

Reply:
You are correct -- I blindly followed the 'DeviceClass'. Will fix.